main/realmain.cc

```diff
@@ -723,8 +723,8 @@ int realmain(int argc, char *argv[]) {
             cache::maybeCacheGlobalStateAndFiles(OwnedKeyValueStore::abort(move(kvstore)), opts, *gs, *workers,
                                                  nonPackageIndexed);
 
-            // Second run: all the other files (the packageDB shouldn't change)
-            pipeline::package(*gs, absl::Span<ast::ParsedFile>(nonPackageIndexed), opts, *workers);
+            // Now validate all the other files (the packageDB shouldn't change)
```

Comment:
Is there some way that we can enforce this constraint?  I imagine that it was "enforced" in the sense that any changes that _did_ occur would just be covered up by running the equivalent of `buildPackageDB` a second time, but it would be nice if we could provided some sort of verification in debug builds.

Reply:
The only thing that would change the package db would be the presence of `__package.rb` files given to this function, right? if so, wouldn't the enforce about not seeing `__package.rb` files in this function be sufficient?